lib/scopes/form526_submission_state.rb

```diff
@@ -38,6 +44,12 @@ module Form526SubmissionState
         where.not(backup_submitted_claim_id: nil)
              .where(backup_submitted_claim_status: backup_submitted_claim_statuses[:rejected])
       }
+      scope :pdf_found, lambda {
```

Comment:
One non-blocking comment: it seems like this scope only applies to Lighthouse submissions, is there any reason not to specify that in the name as was done with EVSS in `accepted_to_evss_primary_path`? Maybe because we're eventually deprecating EVSS so it won't be necessary to specify this is Lighthouse only?

I may be missing something but yeah I was a little confused this name didn't include `lighthouse` so not sure if there's a reason for that or maybe it's worth tweaking the name to make the distinction clearer?

Reply:
That's great feedback. I'm going to change that now. Thanks @NB28VT!